### ISSUE TITLE
Copy to clipboard button obscured on single-line code blocks

### ISSUE THREAD

**Issue by pjohnst5** [EnlighterJS/Plugin.WordPress#342]
**Describe the bug**
The "copy to clipboard" button is obscured on single-lines of code

**WordPress Editing Mode**
* Gutenberg

**Frontend or Backend Issue**
Frontend

**To Reproduce**
Insert single line of code in Guttenberg  Enlighter code block
![image](https://user-images.githubusercontent.com/35265851/147680276-d31e376d-5b87-43d7-b215-9bba86edd745.png)

**Expected behavior**
Expect the copy to clipboard to not be obscured

**Screenshots**
**live repro** https://coolstercodes.com/azure-container-instance-with-load-balancer/
![image](https://user-images.githubusercontent.com/35265851/147680363-29a389f6-505c-4dfd-a708-85ba48366e49.png)

**Example code**
```
<!-- wp:enlighter/codeblock {"language":"md"} -->
<pre class="EnlighterJSRAW" data-enlighter-language="md" data-enlighter-theme="" data-enlighter-highlight="" data-enlighter-linenumbers="" data-enlighter-lineoffset="" data-enlighter-title="" data-enlighter-group="">netsh trace start hello world</pre>
<!-- /wp:enlighter/codeblock -->
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version 96.0.4664.110 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
N/A

**Additional context**
Basically the copy to clipboard button is slightly out of view on single line code blocks.
Is there any way I can make it appear on the line without being hidden partially?

**Comment by AndiDittrich** [comment]
see https://github.com/EnlighterJS/documentation/blob/master/Tweaks.md#set-minimal-height-for-single-line-codeblocks

**Comment by pjohnst5** [comment]
Thanks
